components/script/dom/html/htmliframeelement.rs

```diff
@@ -93,6 +95,8 @@ pub(crate) struct HTMLIFrameElement {
     sandbox_allowance: Cell<Option<SandboxAllowance>>,
     load_blocker: DomRefCell<Option<LoadBlocker>>,
     throttled: Cell<bool>,
+    #[ignore_malloc_size_of = "Rc"]
```

Comment:
We can use #[conditional_malloc_size_of] instead for all of these.

Reply:
Will update after my holiday.

Reply:
Should be fixed now.